#goal
nav_msgs/Path nav_path
---
#result
nav_msgs/Path nav_path
geometry_msgs/Pose end_pose
---
#feedback
int32 last_full_pose
